src/components/screens/vacation-requests-screen.tsx

```diff
@@ -39,6 +43,10 @@ const VacationRequestsScreen = () => {
     adminMode ? allVacationRequestStatusesAtom : vacationRequestStatusesAtom
   );
   const [loading, setLoading] = useState(false);
+  const persons = useAtomValue(personsAtom);
+  const loggedInPerson = persons.find((person: Person) => person.keycloakId === userProfile?.id);
+  const { personsApi } = useApi();
+  const setPersons = useSetAtom(personsAtom);
```

Comment:
persons and setPersons could be combined into one tuple with useAtom()

Reply:
Combined.